toonz/sources/toonz/startuppopup.cpp

```diff
@@ -369,21 +387,17 @@ void StartupPopup::showEvent(QShowEvent *) {
       m_recentNamesLabels[i] = new StartupLabel(justName, this, i);
       m_recentNamesLabels[i]->setToolTip(
           name.remove(0, name.indexOf(" ") + 1));  // remove "#. " prefix
-      m_recentSceneLay->addWidget(m_recentNamesLabels[i], i, Qt::AlignTop);
+      m_recentSceneLay->addWidget(m_recentNamesLabels[i], 0, Qt::AlignTop);
       i++;
     }
-    m_recentSceneLay->addStretch(100);
   }
-
   bool ret = true;
   for (int i = 0; i < m_recentNamesLabels.count() && i < 7; i++) {
```

Comment:
Good catch.  I'm guessing I put that there before I actually knew how many recent scenes would fit there.